hugegraph-core/src/main/java/com/baidu/hugegraph/type/define/DataType.java

```diff
@@ -49,6 +49,9 @@
     private final byte code;
     private final String name;
     private final Class<?> clazz;
+    private final static String POSITIVE_INFINITY = "Infinity";
+    private final static String NEGATIVE_INFINITY = "-Infinity";
+    private final static String NaN = "NaN";
```

Comment:
use ImmutableSet and contains()?

Reply:
seems no benefit if we don't have method like `CollectionUtils.containsAny()`?